### PR TITLE
update readme and code samples

### PR DESCRIPTION
**Pull request by mdumandag**
Updated Readme to get rid of the repetitive names. Renamed `hazelcast-client-sample.json` as `hazelcast-client-full.json` and  added missing properties. Also, changed some single quotes to double quotes in the code samples.

### CODE REVIEW

**Comment by mdumandag** [comment]
verify

File: README.md [hunk 1]
```diff
@@ -2152,13 +2157,14 @@ topic.addMessageListener(function (message) {
     console.log(message.messageObject);
 });
 
-var engineer = {
-    name: 'Furkan Senharputlu',
-    age: 23,
-    field: 'Computer Engineering',
-    university: 'Bogazici University'
+var movie = {
```
Comment (furkansenharputlu) [inline README.md]: I wrote this documentation, lots of engineers use their names in tests, docs etc. There is no problem with this. Also, this change doesn't add extra thing. Is it a must to use foreign names ? I wanted to use TURKISH names. See hazelcast docs using `ali`, `veli`: https://docs.hazelcast.org/docs/3.11.2/manual/html-single/index.html#using-map-with-java-client. 

The purpose of this PR is not an enhancement or fix or something..

Comment (mdumandag) [reply README.md]: This pr is meant to check the readme and update the inputs used in examples. I mean, there were more than 10 occurrences of `Furkan` so I changed them with the inputs used in other clients (mixture of python and cpp). I didn't mean to disregard your efforts

Comment (furkansenharputlu) [reply README.md]: Don't forget that this is the base documentation that is copy-pasted to all clients and C++ mostly uses Node.js client examples. Also, these examples aren't used in Python too. For example, I couldn't see `Christopher Nolan`. Anyway, I didn't mean my efforts are disregarded. My commits are still there. I just wanted to say there is no problem with TURKISH names.

Comment (mdumandag) [reply README.md]: I also think that there is no problem with the Turkish names. This sample came to mine mind while editing the readme for the reason I wrote above and I just changed it to that.

As long as the readme serves it purpose, I think it is okay. Whether we used Turkish names or English names does not matter

Comment (furkansenharputlu) [reply README.md]: Yes, you're right so there is no need to make this change because the existing one does the job.

**Pull request merged**